addons/l10n_pk_edi/models/product_product.py

```diff
@@ -0,0 +1,60 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    l10n_pk_edi_sale_type = fields.Many2one('product.sale.type', 'Sale Type',
+        compute='_compute_l10n_pk_edi_sale_type', required=True, readonly=False, store=True,
+    )
+    l10n_pk_edi_schedule_code = fields.Many2one('product.schedule.code', 'Schedule Code',
+        compute='_compute_l10n_pk_edi_schedule_code', required=True, readonly=False, store=True,
+    )
+
+    @api.depends('type')
+    def _compute_l10n_pk_edi_sale_type(self):
+        for line in self:
+            code_map = {'consu': 'T1000017', 'service': 'T1000018'}
+            code = code_map.get(line.type, 'T1000083')
+            line.l10n_pk_edi_sale_type = self.env['product.sale.type'].search(
+                [('sale_type', '=', code)]
+            ).id
+
+    @api.depends('type')
+    def _compute_l10n_pk_edi_schedule_code(self):
+        for line in self:
+            line.l10n_pk_edi_schedule_code = self.env['product.schedule.code'].search(
+                [('schedule_code', '=', "S1000012")]
+            ).id
+
+
+class ProductSaleType(models.Model):
+    _name = "product.sale.type"
```

Comment:
```suggestion
class L10nPkProductType(models.Model):
    _name = "l10n.pk.product.type"
```
Different file would be much easier to find models like this